.github/workflows/pypi_build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build for PyPi
+
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Setup Poetry
+        uses: Gr1N/setup-poetry@v7
+
+      - name: Install Packages
+        run: poetry install --no-dev
+
+      - name: Build
+        run: poetry build
+
+      - name: Publish
+        run: poetry publish -u ${{ secrets.PYPI_USER}} -p ${{ secrets.PYPI_PASS}}
```

Comment:
consider masking https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#masking-a-value-in-log

Reply:
Secrets are redacted from logging by default. https://docs.github.com/en/actions/security-guides/encrypted-secrets#accessing-your-secrets